staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -123,8 +123,7 @@ func WithAudit(handler http.Handler, sink audit.Sink, policy audit.PolicyRuleEva
 func evaluatePolicyAndCreateAuditEvent(req *http.Request, policy audit.PolicyRuleEvaluator) (*audit.AuditContext, error) {
 	ctx := req.Context()
 	ac := audit.AuditContextFrom(ctx)
-	if ac == nil {
-		// Auditing not enabled.
+	if !ac.Enabled() {
```

Comment:
with the current approach `ac.Enabled` should always return true here, it could also be a little confusing to the reader - the function that is evaluating audit policy is checking whether policy has been evaluated already, 
we could remove the check entirely, or bring back the nil check here, thoughts?

Reply:
Yeah, that makes sense. Done.